src/pages/CarsList/CarsList.tsx

```diff
@@ -86,11 +91,40 @@ function CarsList() {
           Hatchback
         </Button>
       </ButtonGroup>
+      {carType === 4 && (
+        <p className="result-count" data-testid="resultcount">
+          {
+            cars?.filter((car: CarDetails) =>
+              car.name
+                .toLowerCase()
+                .includes(carNameToBeSearched?.toLowerCase())
+            )?.length
+          }{" "}
+          total results
+        </p>
+      )}
       <Container className="carsContainer">
         {cars?.length === 0 && <div id="nocars">No Cars Available</div>}
-        {cars?.map((car: CarDetails) => (
-          <CarCard key={car.id} car={car} />
-        ))}
+        {carType === 4 &&
+          cars?.filter((car: CarDetails) =>
+            car.name.toLowerCase().includes(carNameToBeSearched?.toLowerCase())
+          )?.length === 0 && (
+            <div className="sorry-message">
+              <p> Sorry, the car you have searched is not available</p>
+              <img src={sademoji} alt="coming soon..." width={200}></img>
```

Comment:
why does the sad emoji has atl text of coming soon? Alt ttext of images should just be defining what's in the image. 
I would suggest changing the alt text to "Sad smiley face"